jfr-daemon/src/main/java/com/newrelic/jfr/daemon/JFRUploader.java

```diff
@@ -116,10 +116,12 @@ RecordingFile openRecordingFile(Path file) {
 
   void deleteFile(Path dumpFile) {
     try {
+      logger.debug("Attempting to delete file/path: {}", dumpFile.toString());
```

Comment:
I'd add the file/directory attributes to the log also.

Reply:
Added r/w/x permissions to log on error